.github/workflows/tests.yml

```diff
@@ -80,7 +80,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18, 20]
+        node-version: [18, 20, 22]
```

Comment:
same here

Reply:
Did it here [0404a5e](https://github.com/KILTprotocol/sdk-js/pull/871/commits/0404a5e49a9148c1b75bbff5dc392962725c7e86).